lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts

```diff
@@ -50,4 +50,41 @@ export class LegalHoldService {
             )
         );
     }
+
+    /**
+     * Assign a child of a hold with id holdId.
+     *
+     * @param ids The one element list of manage hold Ids
+     * @param holdId The Id of the hold to which children will be added
+     * @returns Entry with hold Id
+     */
+    assignHold(ids: [string], holdId: string): Observable<{ entry: { id: string } }> {
+        return from(this.legalHoldApi.assignHold(ids, holdId)).pipe(
+            map((response) => ({
+                entry: { id: response.entry.id }
+            }))
+        );
+    }
+
+    /**
+     * Assign a child of a hold with id holdId.
+     *
+     * @param ids The list of manage hold Ids
+     * @param holdId The Id of the hold to which children will be added
+     * @returns List of assigned holds Hold[]
+     */
+    assignHolds(ids: string[], holdId: string): Observable<HoldPaging> {
+        return from(this.legalHoldApi.assignHolds(ids, holdId));
+    }
+
+    /**
+     * Unassign the relationship between a child with id holdChildId and a parent hold with id holdId.
+     *
+     * @param holdId The handled hold Id
+     * @param holdChildId The Id of the child hold which is deleted
+     * @returns Empty response
+     */
+    unassignHold(holdId: string, holdChildId: string): Observable<undefined> {
```

Comment:
```suggestion
    unassignHold(holdId: string, holdChildId: string): Observable<unknown> {
```

Reply:
Should rather be `void` if API returns nothing